tools/nodetool

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+SCRIPT_PATH="$(dirname $(realpath $0))"
+
+# We want this script to work OOTB both in-repo and when installed, while
+# allowing the user to override this behaviour.
+# So try to come up with meanigful default according to the detected location,
+# but allow for override via env variables.
+if [[ "$SCRIPT_PATH" == "/opt/scylladb/bin" ]]
+then
+    DEFAULT_SCYLLA_PATH="/opt/scylladb/bin/scylla"
+    DEFAULT_NODETOOL_PATH="/opt/scylladb/bin/nodetool"
+else
+    DEFAULT_SCYLLA_PATH="$(realpath ${SCRIPT_PATH}/../build/dev/scylla)"
+    DEFAULT_NODETOOL_PATH="${SCRIPT_PATH}/java/bin/nodetool"
+fi
+
+SCYLLA_PATH="${SCYLLA_PATH:-$DEFAULT_SCYLLA_PATH}"
+NODETOOL_PATH="${NODETOOL_PATH:-$DEFAULT_NODETOOL_PATH}"
+
+# Capture the output of scylla-nodetool, as it might just print an error about
+# the command not existing.
```

Comment:
This is fine if it's true that we don't have any nodetool command that prints its output slowly. Is it true?

Another simpler alternative would be to make sure that "scylla nodetool" could - perhaps if given a special "-q" (quiet)  option, will not print anything in case the command is not supported, and just return the 100 immediately. A lot of commands have such a "-q" option.

Reply:
or, feed the tool with `$* --help` to check if the arguments are supported. so if the command line is not accepted, it errors out, we fall back the java-based nodetool, if scylla returns status code of 0 (i assume that `--help` does not return non-zero status code), we feed it `$*` without `--help`.